howto/manage/setup-tls-for-appliance.md

```diff
@@ -0,0 +1,81 @@
+The Anbox Cloud Appliance uses the self-signed certificate to provide HTTPS services hence it is considered to be used for development or testing purposes in the majority of cases due to the security risks. In case you want to use it for production, you will need to generate a valid SSL certificate and install it to the Anbox Cloud Appliance.
+
+[note type="information" status="Note"]This tutorial assumes you have Anbox Cloud Appliance installed. If you haven't, please follow the [instruction](https://discourse.ubuntu.com/t/install-the-anbox-cloud-appliance/22681) to do so.[/note]
+
+## Add a DNS Record
+
+Setting up DNS redirection depends on your DNS provider. Refer to the documentation of your provider to create a DNS record pointing to the IP/DNS of the AWS instance where the Anbox Cloud appliance is running on.
+
+## Configure the location
+
+Configure the location for the appliance using the created DNS name.
+
+    sudo snap set anbox-cloud-appliance experimental.location=<your DNS name>
+
+## Generate an SSL certificate
+
+There are many ways to create a valid SSL certificate. It's recommended to use [Let's Encrypt](https://letsencrypt.org/) to generate a free SSL certificate.
+
+First, connect and SSH into your appliance instance, and install the `certbot` snap:
+
+    sudo snap install --classic certbot
+
+Before generating your certificate using `certbot`, you need to stop the traefik service from listening on port 80 for the certificate creation.
```

Comment:
This will conflict with the renewal process and will prevent an automatic renewal unless we instruct the renewal process to include stop/start traefik.

Reply:
See `Renew an SSL certificate` below